src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -1129,16 +1194,35 @@ void MKLDNNDeformableConvolutionNode::prepareParams() {
 
     updatePadding();
 
-    std::vector<std::shared_ptr<BlockedMemoryDesc>> descVector {
-        getParentEdgeAt(DATA_ID)->getMemory().GetDescWithType<BlockedMemoryDesc>(),
-        getParentEdgeAt(OFF_ID)->getMemory().GetDescWithType<BlockedMemoryDesc>(),
-        getParentEdgeAt(WEI_ID)->getMemory().GetDescWithType<BlockedMemoryDesc>()
+    DefConvKey key = {
+        srcMemPtr->GetDescWithType<DnnlMemoryDesc>(),
+        offMemPtr->GetDescWithType<DnnlMemoryDesc>(),
+        weiMemPtr->GetDescWithType<DnnlMemoryDesc>(),
+        dstMemPtr->GetDescWithType<DnnlMemoryDesc>(),
+        defConvAttr.stride,
+        defConvAttr.dilation,
+        defConvAttr.padL,
+        getSelectedPrimitiveDescriptor()->getImplementationType()
     };
 
-    if (withModulation) {
-        descVector.push_back(getParentEdgeAt(MOD_ID)->getMemory().GetDescWithType<BlockedMemoryDesc>());
-    }
-    descVector.push_back(getChildEdgesAtPort(0)[0]->getMemory().GetDescWithType<BlockedMemoryDesc>());
+    auto builder = [&](const DefConvKey& key) -> std::shared_ptr<DefConvExecutor> {
```

Comment:
you need to use all values necessary to create primitive from key only, and also remove capturing by reference from lambda please.

Reply:
Done